pkgs/by-name/bu/bustle/package.nix

```diff
@@ -32,6 +36,14 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-r29Z+6P+yuCpOBUE3vkESd15lcGXs5+ZTBiQ9nW6DJ4=";
   };
 
+  # Use gettext from nixpkgs instead of the vendored one, which does not build on Darwin with clang 16.
+  env = {
+    GETTEXT_BIN_DIR = lib.getBin buildPackages.gettext;
+    GETTEXT_INCLUDE_DIR = lib.getDev gettext;
+    GETTEXT_LIB_DIR = lib.getLib gettext;
+    GETTEXT_SYSTEM = true;
```

Comment:
These should be only needed on Darwin?

https://gitlab.gnome.org/World/bustle/-/blob/816488be5d81890e167b844460078a463b18eb84/Cargo.toml#L13
https://github.com/gettext-rs/gettext-rs/blob/master/gettext-sys/build.rs#L88-L91

Reply:
It seemed harmless to do unconditionally, but I can make it conditional on Darwin. I’ll do the same for the other PRs that also fix the same issue for other packages.